### DISCUSSION TITLE
Napi forgalmi jelentés tárolása

### DISCUSSION THREAD

**Discussion by MicraMetripond2** [nav-gov-hu/eRECEIPT#14] [Kérdések és válaszok]
A Napi forgalmi jelentést rendszeresen használják a könyvelésnél. A mostani terv szerint ezt az adóügyi bizonylatot nem kötelező nyomtatni és a nyugtatárba sem küldendő.
A NAV fog ezekről a jelentésekről adatot szolgáltatni?

**Comment by renced42** [comment]
Kedves @MicraMetripond2 
A pénztárgépnek kell adatot szolgáltatnia erről a bizonylatról, amit a NAV vissza tud mutatni az adózónak. Ez az adatkör nem kerül be a nyugtatárba tekintettel arra, hogy ez nem a vevőre tartozó nyugta.
